src/components/DialogContentContainer/__stories__/DialogContentContainer.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import DialogContentContainer from "../DialogContentContainer";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import DialogContentContainerExample from "./DialogContentContainerExample";
+import "./DialogContentContainer.stories.scss";
+
+<Meta title="Popover/DialogContentContainer" component={DialogContentContainer} />
+
+# DialogContentContainer
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview">
+    <DialogContentContainer>
+      <DialogContentContainerExample />
```

Comment:
I think it may be confusing here to use an example component instead of show the div and p components (developers may think they should know more api's which are not in the docs)

Reply:
API's = props, if someone is looking for more than we are doing something wrong